unimol_tools/pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "unimol_tools"
-version = "0.2.0.post2"
+version = "0.2.1"
```

Comment:
This one is more important than `unimol` so it got a minor bump instead of a path bump? :D